slacker.py

```diff
@@ -9,7 +9,9 @@ def __init__(self, success, files, prefixes, recordcount, validationcount, error
             self.validation = "N/A"
         else:
             self.validation = "FAILED"
-        self.files = files
+        self.file_count = len(files)
+        self.files_string = "\n".join(files)
+        self.files_string = self.files_string[:2971] # 3000 character maximum for Slack blocks (29 characters for wrap)
```

Comment:
Let's append something like `... [TRUNCATED LIST]` so we know that it is not a full list.

Reply:
Added.